### PR TITLE
Fixed #1083 - AOR_Reports - Contional date value decrements on every …

### LINKED ISSUE #1083: AOR_Reports - Contional date value decrements on every "update"
When you're pulling a report (avoiding bug #1082 by all using same format) with the conditional date fields set to "value", every time you click update, each of the date fields decrements by 1 day.

Example:
1. Set condition date1: value > 2016-03-01
2. Set condition date2: value > 2016-03-11
3. Hit "update". Proper results are shown
4. Look at dates, date1 now equals 2016-02-29 and date2 now equals 2016-03-10

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

Fixed #1083 - Display date as is - without modifying (adjusting it according to time zone might decrease/increase date)

## How To Test This

1. Set condition date1: value > 2016-03-01
2. Set condition date2: value > 2016-03-11
3. Hit "update". Proper results are shown. Dates remain the same.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/AOW_WorkFlow/aow_utils.php [hunk 1]
```diff
@@ -519,11 +519,9 @@ function getModuleField($module, $fieldname, $aow_field, $view='EditView',$value
             $fieldlist[$fieldname]['value'] = $timedate->to_display_date_time($value, true, true);
         }
         $fieldlist[$fieldname]['name'] = $aow_field;
-    } else if(isset( $fieldlist[$fieldname]['type'] ) && ($fieldlist[$fieldname]['type'] == 'datetimecombo' || $fieldlist[$fieldname]['type'] == 'datetime' || $fieldlist[$fieldname]['type'] == 'date')){
+    } else if (isset($fieldlist[$fieldname]['type']) && ($fieldlist[$fieldname]['type'] == 'datetimecombo' || $fieldlist[$fieldname]['type'] == 'datetime' || $fieldlist[$fieldname]['type'] == 'date')) {
         $value = $focus->convertField($value, $fieldlist[$fieldname]);
-        $fieldlist[$fieldname]['value'] = $timedate->to_display_date($value);
-        //$fieldlist[$fieldname]['value'] = $timedate->to_display_date_time($value, true, true);
-        //$fieldlist[$fieldname]['value'] = $value;
+        $fieldlist[$fieldname]['value'] = substr($value, 0, 10);
```
Comment (gymad) [inline modules/AOW_WorkFlow/aow_utils.php]: If you have to replace to_display_date() function to fixing this issue, it means something wrong with in this function? If it is the case may better practice to fixing it centralized.

Comment (Dillon-Brown) [reply modules/AOW_WorkFlow/aow_utils.php]: @gymad The to_display_date function converts DB date string to local date string but in this case we don't want to do any conversions.

**Comment by Mac-Rae** [comment]
Please rebase this pull request to the hotfix-7.10.x branch as version 7.8.x of SuiteCRM has reached its end of life and therefore will no longer be receiving updates. 

For more information about 7.8.x end of life please visit our [blog post](https://suitecrm.com/7-8-x-end-of-life/)

**Comment by Dillon-Brown** [comment]
Fix is no longer relevent

**Pull request closed without merging**